source/_patterns/03-templates/25-basket.mustache

```diff
@@ -1,53 +1,19 @@
-
-<div id="empty_basket" style="display:none">
-	<p class="bigger">Oops — Your basket is empty!</p>
-	<p>Your basket is empty. To redeem your Kiva Card <a href="{{ lend_link }}">you'll need to select a loan</a> to support.</p>
-</div>
+{{> 02-organisms/60-checkout/02-empty-basket}}
 
 <div id="basket_contents" class="basketContents">
 	<h1>You have {{ count }} loans in your basket</h1>
 
-	<div class="loan-list"></div>
+	<div id="loan_items" class="listItems">
+		{{> 02-organisms/60-checkout/03-loan-item}}
+	</div>
```

Comment:
a way to write collections like this that works for both pattern-lab and marionette is:
```
<ul id="loan_items" class="listItems">
	{#loans}
		<li>
			{{> 02-organisms/60-checkout/03-loan-item}}
		</li>
	{/loans}
</ul>
```
where 'loans' is only defined in a json data file in pattern-lab. When pattern-lab renders this it will render out a list of loan-items, and when a marionette collection view renders this it skips everything in the #loans block, rendering out an empty ul that then is used by the collection renderer to display the list.